### ISSUE TITLE
Experiment plan

### ISSUE THREAD

**Issue by john-mai-2605** [Seondong/Customs-Fraud-Detection#88]
Building experiment plan and realize with bash code

**Comment by john-mai-2605** [comment]
Main data: t-data
Exp 1: Tuning adahybrid - By independent search or grid search (grid might be too time-consuming)
Hyperparam: Level of randomness, learning rate, decay rate
Exp 2: Main exp: data-t, xgb/hybrid
Strategies: pot, pv, ada, pot+ada, csi, (maybe: csi+ada)
Hypothesis: Approaching or outperforming hybrid
ada+pot > ada, pot > best hybrid
Exp 3: Ablation: different data, DATE/bATE on data-t
